### PR TITLE
upgrade babel-core to 6.0.2

### PR DESCRIPTION
**Pull request by underbyte**

### CODE REVIEW

**Comment by sindresorhus** [comment]
Same comment as https://github.com/babel/gulp-babel/pull/46. Tests and example needs updating.

**Comment by underbyte** [comment]
@sindresorhus let me know if you need any other changes

File: test/test.js [hunk 1]
```diff
@@ -13,5 +14,14 @@ exports.babel = {
 		test.deepEqual(json.file, 'fixture-compiled.js');
 		test.ok(/\/\/# sourceMappingURL=fixture-compiled.js.map\n$/.test(code));
 		test.done();
+	},
+	transpile: function (test) {
+		var res = babel.transformFileSync('test/fixture.js', {
+			presets: ['babel-preset-es2015']
+		});
```
Comment (sindresorhus) [inline test/test.js]: I don't see the point of this test. You're just testing Babel itself. I meant use the preset option in the original test.

Comment (underbyte) [inline test/test.js]: I can remove this test but I think updating the grunt file should sufficient then.

**Comment by sindresorhus** [comment]
Thanks :)

**Pull request merged**